components/02-molecules/alert/_yds-alert.scss

```diff
@@ -14,6 +15,7 @@ $alert-themes: map.deep-get(sass-tokens.$tokens, color, alert);
   z-index: 3;
   background-color: var(--color-alert-background);
   color: var(--color-alert-text);
+  opacity: 1;
```

Comment:
Looks like you're animating the opacity of the outer-wrapper, and the "inner" element. I think we can consolidate that to just the outer-wrapper.

Reply:
@ModulesUnraveled This is because I want to animate both the wrapper (`opacity`) with the bg-color and the inner text content (`height`). I want the inner content's height to animate out more gracefully than if we had animations only on the wrapper.

Are you saying that I should apply ` @include tokens.animate(all, $duration: $alert-fade-speed);` to the `.alert` element and remove the `inner` animations? If we do that, the alerts simply collapse and the animation is not noticeable. I'm totally open to whatever suggestions you have, but I'm not clear on how we can simplify this while retaining the animation currently in place. Please let me know where you see opportunities for improvement. 

I did update where the animation was being applied - move to `alert` and `alert__inner` instead of `[data-alert-state='dismissed']`.